starter-project-web-react/pages/postBlog.jsx

```diff
@@ -0,0 +1,14 @@
+
+import CreateBlog from '../components/creatBlog';
+
+
+const AddBlog = ()=>{
+    return (
+        <>
```

Comment:
I am not sure what this is meant to be
if it is not needed, better to get rid of it

Reply:
![image](https://user-images.githubusercontent.com/61737042/165284104-0ba082c8-c267-4d77-803e-8bd5f38874aa.png)